openpower/patches/p8dtu-patches/hostboot-p8/0011-always-return-success-for-VPD-write.-patch-github-is.patch

```diff
@@ -0,0 +1,28 @@
+From ed240ea8835c61ea639737eafeba87ec6070ba0d Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Wed, 27 Jul 2016 16:22:31 -0700
+Subject: [PATCH 11/14] always return success for VPD write. patch github issue
+ #114 for P8DTU.
```

Comment:
This issue https://github.com/open-power/op-build/issues/114 ? That doesn't seem right...

If it's https://github.com/open-power/supermicro-openpower/issues/114 instead, please mention it.

@dcrowell77 does this look right? Is just ignoring error messages here a good idea?

Reply:
The code path that was changed should never run in OpenPOWER.  There is a check right above this in the code that skips the function if we're not on a FSP.

Reply:
@dcrowell77 Okay... so it sounds like this patch isn't needed then as the code can't possibly be run?

Reply:
Yes, I believe that is the case.

Reply:
Cool.  Will not include this patch for now.

Reply:
Delete this patch.